source/write/run-command.txt

```diff
@@ -0,0 +1,182 @@
+.. _php-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: administration, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+}
+to run a database command. You can use database commands to perform a
+variety of administrative and diagnostic tasks, such as fetching server
+statistics, initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Library Methods to Database Commands
+
+   The library provides wrapper methods for many database commands.
+   We recommend using these methods instead of executing database
+   commands when possible.
+   
+   To perform administrative tasks, use the :mongosh:`MongoDB Shell </>`
+   instead of the {+php-library+}. Calling the ``db.runCommand()``
+   method inside the shell is the preferred way to issue database
+   commands, as it provides a consistent interface between the shell and
+   drivers or libraries.
+
+.. _php-execute-command:
+
+Execute a Command
+-----------------
+
+To run a database command, you must specify the command and any relevant
+parameters in a command document, then pass the command document to the
+``command()`` method. This method returns a ``Cursor`` object.
+
+The following code shows how you can use the ``command()``
+method to run the ``hello`` command, which returns information about
+the current member's role in the replica set, on a database:
```

Comment:
The code example is showing the `ping` command. Update the description or update the code example to run 'hello' instead. (`ping` description in suggestion taken from the server manual)
```suggestion
The following code shows how you can use the ``command()``
method to run the ``ping`` command, which is a no-op command used to test whether a server is responding:
```

Reply:
oops let me rework!